examples/cluster/plot_choose_nb_cluster.py

```diff
@@ -0,0 +1,89 @@
+from __future__ import division
```

Comment:
You have to put a docstring explaining the example.
Then put a `print(__doc__)`.
You can take as example the file : `plot_cluster_comparison.py`